server/graphql/v2/object/AccountPermissions.ts

```diff
@@ -18,7 +18,7 @@ const AccountPermissions = new GraphQLObjectType({
       type: new GraphQLNonNull(Permission),
       description: 'Whether the current user can mark this order as expired',
       resolve(collective, _, req: express.Request): PermissionFields {
-        return { allowed: checkScope(req, 'host') && req.remoteUser.isHostAdmin(collective) };
```

Comment:
Such a function `isHostAdimin` seems not to exist. Therefor I am suggesting this alternative approach. 

cc: @Betree